redash/query_runner/clickhouse.py

```diff
@@ -10,6 +11,12 @@
 logger = logging.getLogger(__name__)
 
 
+def split_multi_query(query):
+    queries = re.split(r";[\s;]*$", query, flags=re.MULTILINE)
+
+    return [q.strip() for q in queries if q and not q.isspace()]
+
+
```

Comment:
Generally it's not a good idea to split statements with regex. We already have a `sqlparse` dependency that could be used for this purpose. I recently implemented similar feature for Databricks query runner (getredash/redash#5058), you can check my code for the reference

Reply:
Wow, it took me a while to fix that :) Done!